tools/release_do_nothing.py

```diff
@@ -10,879 +10,859 @@
 
-    message = (
-        f"Follow the Pull Request process to get {release_strings.tag} branch "
-        f"merged into upstream/{upstream_branch} .\n"
-        "Specific conda-forge guidance will be automatically given once the "
-        "PR is created."
-    )
-    _wait_for_done(message)
+        def validate(sha256_string: str) -> str:
+            try:
+                sha_len = int(sha256_string, 16).bit_length() + 1
+            except ValueError:
+                sha_len = 0
 
-    message = (
-        f"Confirm that {release_strings.release} appears in this list:\n"
-        "https://anaconda.org/conda-forge/iris/files"
-    )
-    _wait_for_done(message)
+            if sha_len != 256:
+                self.report_problem("Invalid SHA256 hash. Please try again ...")
+            else:
+                return sha256_string
 
-    if not is_release_candidate:
         message = (
-            f"Confirm that {release_strings.release} is displayed on this "
-            "page as the latest available:\n"
-            "https://anaconda.org/conda-forge/iris"
+            f"Visit the below and click `view hashes` for the Source Distribution"
+            f"(`.tar.gz`):\n"
+            f"https://pypi.org/project/scitools-iris/{self.strings.release}#files\n"
+        )
+        self.set_value_from_input(
+            key="sha256",
+            message=message,
+            expected_inputs="Input the SHA256 hash",
+            post_process=validate,
         )
-        _wait_for_done(message)
-
-    if is_release_candidate:
-        channel_command = " -c conda-forge/label/rc_iris "
-    else:
-        channel_command = " "
-    message = (
-        "Confirm that conda (or mamba) install works as expected:\n"
-        f"conda create -n tmp_iris{channel_command}iris={release_strings.release};\n"
-        f"conda remove -n tmp_iris --all;"
-    )
-    _wait_for_done(message)
-
-
-def update_links(release_strings: ReleaseStrings) -> None:
-    _break_print("Link updates ...")
 
-    message = (
-        "Revisit the GitHub release:\n"
-        f"https://github.com/SciTools/iris/releases/tag/{release_strings.tag}\n"
-        "You have confirmed that Read the Docs, PyPI and conda-forge have all "
-        "updated correctly. Include the following links in the release "
-        "notes:\n\n"
-        f"https://scitools-iris.readthedocs.io/en/{release_strings.tag}/\n"
-        f"https://pypi.org/project/scitools-iris/{release_strings.release}/\n"
-        f"https://anaconda.org/conda-forge/iris?version={release_strings.release}\n"
-    )
-    _wait_for_done(message)
+        message = (
+            "Confirm that pip install works as expected:\n"
+            "conda create -y -n tmp_iris pip cf-units;\n"
+            "conda activate tmp_iris;\n"
+            f"pip install scitools-iris=={self.strings.release};\n"
+            'python -c "import iris; print(iris.__version__)";\n'
+            "conda remove -n tmp_iris --all;\n"
```

Comment:
This requires a `"conda deactivate"` to work

Reply:
0f8721b47